k8s.go

```diff
@@ -388,3 +417,7 @@ func (c *Client) ReplaceKubernetesNode(clusterID, nodePoolID, nodeID string) (*h
 	h := rsp.Header()
 	return &h, validateResponse(rsp, http.StatusAccepted)
 }
+
+func (a AutoScaling) Enabled() bool {
```

Comment:
Please add a tests where autoscaling is not included in the JSON and then call `Enabled()`. Either make this a pointer receiver or use an Autoscaling struct value in `KubernetesNodePoolProperties`.